src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Color color = colorSupplier.getRandomColor();
+    private int number = new Random().nextInt(100);

```

Comment:
100 is magic number
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#all-magic-numbers-in-your-code-should-be-constants